### DISCUSSION TITLE
I have an error for using joins (SQLSTATE[23000]: Integrity constraint violation: 1052 Column 'created_at' in where clause is ambiguous).

### DISCUSSION THREAD

**Discussion by mani1430** [Power-Components/livewire-powergrid#468] [Q&A]
![image](https://user-images.githubusercontent.com/90332512/172305759-ebe90906-5a84-4b8f-8984-86844b3a5a1d.png)
I have an error for selecting flatpicker. please give me a solution.

**Comment by codemasterjoel** [comment]
where it says created_at place the name of the table followed by a point example erp_customer_masters.created_at

**Reply by Kakad3v** [reply]
This is the solution
